src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -309,11 +312,22 @@ public record Application
   public IEnumerable<CertificationType> CertificationTypes { get; set; } = Array.Empty<CertificationType>();
   public IEnumerable<Transcript> Transcripts { get; set; } = Array.Empty<Transcript>();
   public IEnumerable<WorkExperienceReference> WorkExperienceReferences { get; set; } = Array.Empty<WorkExperienceReference>();
+  public IEnumerable<ProfessionalDevelopment> ProfessionalDevelopments { get; set; } = Array.Empty<ProfessionalDevelopment>();
   public ApplicationStatus Status { get; set; }
   public PortalStage Stage { get; set; }
   public IEnumerable<CharacterReference> CharacterReferences { get; set; } = Array.Empty<CharacterReference>();
+  public string? ExplanationLetter { get; set; }
+  public OneYearRenewalexplanations OneYearRenewalexplanation { get; set; }
+}
+public record ProfessionalDevelopment([Required] string? CertificationNumber, [Required] DateTime CertificationExpiryDate, [Required] DateTime DateSigned, [Required] string? CourseName, [Required] string? OrganizationName, [Required] DateTime StartDate, [Required] DateTime EndDate)
```

Comment:
question, for the [Required] string fields - does that mean string? -> string since it can't be null.